content/en/llm_observability/_index.md

```diff
@@ -55,10 +52,18 @@ Automatically scan and redact any sensitive data in your AI applications and ide
 
 {{< img src="llm_observability/prompt_injection.png" alt="An example of a prompt-injection attempt detected by LLM Observability" style="width:100%;" >}}
 
-<div class="alert alert-warning">By using LLM Observability, you acknowledge that Datadog is authorized to share your company's data with OpenAI LLC for the purpose of providing and improving LLM Observability. OpenAI will not use your data for training or tuning purposes. If you have any questions or want to opt out of features that depend on OpenAI, reach out to your account representative.</div>
+## Use integrations with LLM Observability
```

Comment:
could we add a bigger line break before this header? looks a little compact

Reply:
```suggestion
<br>
## Use integrations with LLM Observability
```